static/app/views/releases/releaseBubbles/useReleaseBubbles.tsx

```diff
@@ -335,7 +334,6 @@ interface UseReleaseBubblesParams {
   releases?: ReleaseMetaBasic[];
 }
 export function useReleaseBubbles({
-  chartRef,
```

Comment:
This is redundant since we were already creating a ref callback with `createReleaseBubbleHighlighter` -- no need to explicitly pass this.